src/flex-example-app-android.xml

```diff
@@ -79,6 +91,18 @@
 						</intent-filter>
 					</receiver>
 
+
+                    <receiver
+                        android:name="com.inmobi.commons.analytics.androidsdk.IMAdTrackerReceiver"
```

Comment:
Actually it worked fine for me, but I removed because no notes about it in docs.